diff --git a/dist/build/webpack/plugins/next-font-manifest-plugin.js b/dist/build/webpack/plugins/next-font-manifest-plugin.js
index 1752d5b645d48ec276d04669486e16a849aaac24..c6debdcafe601dc1d6cbfb6e9de96e7dda43d32b 100644
--- a/dist/build/webpack/plugins/next-font-manifest-plugin.js
+++ b/dist/build/webpack/plugins/next-font-manifest-plugin.js
@@ -57,7 +57,7 @@ class NextFontManifestPlugin {
                     // After all modules are created, we collect the modules that was created by next-font-loader.
                     (0, _utils.traverseModules)(compilation, (mod, _chunk, chunkGroup)=>{
                         var _mod_request;
-                        if (mod == null ? void 0 : (_mod_request = mod.request) == null ? void 0 : _mod_request.includes('/next-font-loader/index.js?')) {
+                        if (mod == null ? void 0 : (_mod_request = mod.request) == null ? void 0 : (_mod_request.includes('/next-font-loader/index.js?') || _mod_request.includes('\\next-font-loader\\index.js?'))) {
                             var _mod_buildInfo;
                             if (!((_mod_buildInfo = mod.buildInfo) == null ? void 0 : _mod_buildInfo.assets)) return;
                             const chunkEntryName = (appDirBase + chunkGroup.name).replace(/[\\/]/g, _path.default.sep);
